README.md

```diff
@@ -15,12 +15,14 @@ QUICKINSTALL:
 
 ```
 # cd espressopp
-# cmake .
-# make
+# cmake -DEXTERNAL_BOOST=OFF -DEXTERNAL_MPI4PY=OFF .
+# make -j 4

```

Comment:
I thought that modern computers have more than one CPU and `make -j 4` will be definitely faster than `make`. I've heard that sometimes one goes overboard if using simply `-j` but have no proof of it. Therefore I've added a comment later saying one can adjust the number. 

@govarguz  what do you think?